### ISSUE TITLE
5 new investigators cards VS chaos

### ISSUE THREAD

**Issue by DgNoah1981** [Kamalisk/arkhamdb#336]
The cards are unavailable for the other investigators when using the chaos options. Seems to stop at the dream eaters campaign box.

**Comment by Kamalisk** [comment]
are they part of your collection or selected in the pack filter?

**Comment by DgNoah1981** [comment]
Yes they are part of the collection. And i am on the mobile version of the site maybe its that idk. But i am testing with Agnes and to never see a card from either Jacqueline or Stella pack appears in any try at all. I’ll go try on the desktop version to see.

Sent from my iPhone

On Sep 5, 2020, at 5:52 AM, Allan Clements <notifications@github.com<mailto:notifications@github.com>> wrote:


are they part of your collection or selected in the pack filter?

**Comment by DgNoah1981** [comment]
Forget it. It was my collection that wasn t updated. I thought it was since the app was.

Sent from my iPhone

On Sep 5, 2020, at 5:52 AM, Allan Clements <notifications@github.com<mailto:notifications@github.com>> wrote:


are they part of your collection or selected in the pack filter?
